### PR TITLE
1.6.0

### PR DESCRIPTION
**Pull request by GhexterCortes**

### CODE REVIEW

**Review by GhexterCortes — COMMENTED** [review]
krazy

File: index.js [hunk 1]
```diff
@@ -12,27 +12,29 @@
     FunctionNames: camelCase
 **/
 
+var configPath = './config/Bot/config.yml';
+var languagePath = './config/Bot/language.yml';
```
Comment (BlobHuman) [inline index.js:16]: Make it constant
```suggestion
const configPath = './config/Bot/config.yml';
const languagePath = './config/Bot/language.yml';
```

File: index.js [hunk 2]
```diff
@@ -12,27 +12,29 @@
...
-const { Config, Language } = require('./scripts/config');
 const Discord = require('discord.js');
 
-// Local actions
+// Local modules
+const { Config, Language } = require('./scripts/config');
 const ScriptLoader = require('./scripts/loadScripts');
 const { SafeMessage, SafeInteract } = require('./scripts/safeActions');
 const CommandPermission = require('./scripts/commandPermissions');
 const MemberPermission = require('./scripts/memberPermissions');
```
Comment (BlobHuman) [inline index.js:29]: Looks better
```suggestion
const { Config, Language } = require('./scripts/config');
const { SafeMessage, SafeInteract } = require('./scripts/safeActions');
const CommandPermission = require('./scripts/commandPermissions');
const MemberPermission = require('./scripts/memberPermissions');
const ScriptLoader = require('./scripts/loadScripts');
```

**Review by BlobHuman — APPROVED** [review]

**Comment by GhexterCortes** [comment]
New year new pr

**Comment by BlobHuman** [comment]
> New year new pr

just commit!

**Comment by GhexterCortes** [comment]
oke

**Review by BlobHuman — APPROVED** [review]

**Comment by BlobHuman** [comment]
incoming bugs 🤣

**Comment by GhexterCortes** [comment]
shut-

**Comment by GhexterCortes** [comment]
stop command disabled  bruh!

**Pull request merged**